src/agents/convenience.jl

```diff
@@ -98,6 +98,11 @@ function init_abm_data_plots!(fig, p, adata, mdata, alabels, mlabels, plotkwargs
             ylims!(ax, Makie.MakieLayout.yautolimits(ax))
         end
     end
+    on(resetclick) do clicks
+        for ax in axs
+            vlines!(ax, [p.s.val], color = "#c41818")
```

Comment:
> #c41818

Was this the colour we've used in the earlier version?

Reply:
Yeah--I just found the last version of the code that had the colored line, and basically tried to modify it as little as possible to make it work after your refactor.